Dockerfile

```
@@ -24,20 +24,76 @@ RUN apt-get update && \
     libssl-dev \
     libxml2-dev \
     zlib1g-dev \
+    libbz2-dev \
+    libenchant-2-dev \
+    libpng-dev \
+    libgmp-dev \
+    libavif-dev \
+    libwebp-dev \
+    libjpeg-dev \
+    libfreetype-dev \
+    libldap-dev \
+    libpq-dev \
+    libxslt1-dev \
+    libzip-dev \
+    libicu-dev \
     bison \
     && \
-    apt-get clean 
+    apt-get clean
 
-RUN git clone --depth=1 --single-branch --branch=frankenphp-8.2 https://github.com/dunglas/php-src.git && \
-    cd php-src && \
-    #export CFLAGS="-DNO_SIGPROF" && \
-    # --enable-embed is only necessary to generate libphp.so, we don't use this SAPI directly
-    ./buildconf && \
+RUN git clone --depth=1 --single-branch --branch=frankenphp-8.2 https://github.com/dunglas/php-src.git
+
+RUN cd php-src && ./buildconf
+
+RUN cd php-src && \
+    `# --enable-embed is only necessary to generate libphp.so, we don't use this SAPI directly` \
     ./configure \
-        --enable-embed=static \
-        --enable-zts \
+        --disable-fpm \
         --disable-zend-signals \
-        --with-config-file-scan-dir=/usr/local/lib/php/ && \
+        --enable-fileinfo \
+        --enable-dom \
+        --enable-filter \
+        --with-libxml \
+        --with-iconv \
+        --enable-bcmath \
+        --enable-calendar \
+        --enable-embed=static \
+        --enable-exif \
+        --enable-ftp \
+        --enable-gd \
+        --enable-intl \
+        --enable-mbstring \
+        --enable-pcntl \
+        --enable-shmop \
+        --enable-sigchild \
+        --enable-soap \
+        --enable-sockets \
+        --enable-sysvmsg \
+        --enable-sysvsem \
+        --enable-sysvshm \
+        --enable-zts \ \
```

```suggestion
        --enable-zts \
```

This double slash did make a problem on my machine.

---

> But I'M also still running into following in a later step:

I had the exact same error, I have no idea on what this means

---

Basically, that libicu and the `-libicu` LDFLAG are required to use this extension I guess.

---

`-licu` does not work, nor `-licu67` after installing `libicu-dev` and `libicu67`.

The `frankenphp.go` file probably needs some rewrite in case we compile every extension 😅

Or maybe we'll drop intl/icu and check later on.

---

You should be able to change the flags by setting the corresponding environment variable (without editing the go file).